gnomad_qc/v3/resources/release.py

```diff
@@ -91,8 +177,8 @@ def hgdp_1kg_subset_annotations(sample: bool = True) -> VersionedTableResource:
     :return: Table resource with sample/variant annotations for the subset
     """
     return VersionedTableResource(
-        CURRENT_RELEASE,
-        {
+        default_version=CURRENT_RELEASE,
```

Comment:
sorry, starting to lose track of what I've already asked. should this be `CURRENT_HGDP_TGP_RELEASE`?

Reply:
I also just noticed there's a `3.1` and `v3.1` here 
![image](https://user-images.githubusercontent.com/19142091/121722153-11b58e80-cab3-11eb-96ee-ca88fd21b388.png)

Reply:
yes I think when Grace made the release the first time she accidentally added the `v` I have this listed on my storage sheet asking if we can just delete